Failing on nightly

`interpolate_idents` is failing on today's nightly. See, e.g., the error from [this Travis build](https://travis-ci.org/ezrosent/allocators-rs/jobs/356020006#L2863):

```text
error[E0599]: no variant named `Underscore` found for type `syntax::parse::token::Token` in the current scope
  --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/interpolate_idents-0.2.2/src/lib.rs:39:52
   |
39 |                         TokenTree::Token(ref span, Token::Underscore) => {
   |                                                    ^^^^^^^^^^^^^^^^^ variant not found in `syntax::parse::token::Token`
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -36,13 +36,6 @@ fn interpolate_idents<'a>(cx: &'a mut ExtCtxt,
                             }
                             new_ident.push_str(&ident.name.as_str());
                         },
-                        TokenTree::Token(ref span, Token::Underscore) => {
-                            match new_span {
-                                Some(ref mut s) => { *s = s.with_hi(span.hi()); },
-                                None => { new_span = Some(span.clone()); },
-                            }
-                            new_ident.push_str("_");
-                        },
                         _ => return None,
                     }
                 }

